javascript/MaterialXView/source/index.js

```diff
@@ -5,6 +5,7 @@
 
 import * as THREE from 'three';
 import { GLTFLoader } from 'three/examples/jsm/loaders/GLTFLoader';
+import { DRACOLoader } from 'three/examples/jsm/loaders/DRACOLoader';
```

Comment:
The draco related code here and in webpack.config were removed on upstream in AcademySoftwareFoundation/MaterialX#806. I think we shouldn't reintroduce it here